src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1443,27 +1443,24 @@ Here is the list of topologies we support:
 
 === Handling failures
 
-It's possible to have errors when your workflow is executed. *ProActive Scheduler* provides several mechanisms to deal
-with exceptional situations in your workflows.
-
-To understand if a task is successfully finished or not the Scheduler checks its *exit code* (for native tasks) or
-*exceptions* (for Java/script tasks). Then, it decides what to do next according to the following parameters:
-
-* *cancelJobOnError* (optional - `false` by default) - defines whether the job must continue when a user exception or
-error occurs during the job process. This property can also be defined in each task. If the value of this property
-is defined at the job level, each cancelJobOnError property of each task will have this value as the default one
-(excepted if this property has been set at the task level). `true` implies for the job to immediately stop
-every remaining running tasks if an error occurs in one of the tasks. It is useful when there is no need to go further after a task failure.
-
-* *restartTaskOnError* (optional - `anywhere` by default) - defines whether tasks that have to be restarted will
-restart on an other resource. Defining this property will set the restartTaskOnError property of each task to this
-value as the default one (excepted if this property has been set at the task level). Possible values are 'anywhere'
-or 'elsewhere' meaning respectively that the concerned task will be restart on any available resources or especially
-on a different one. A task can be restarted when an exception occurred (Java Task) or an error code is returned (Native Task).
-
-* *maxNumberOfExecution* (optional - `1` by default) - defines how many times tasks are allowed to be restarted.
-Defining this property will set the nbMaxOfExecution property of each task to this value as the default one
-(excepted if this property has been set at the task level). The value has to be a non-negative and non-null integer.
+It’s possible to have errors when your job is executed. *ProActive Scheduler* provides several mechanisms to deal with exceptional situations in your jobs.
+If a task contains errors during job execution, the *Proactive Scheduler* automatically restarts this task. The number of automatic restarts is defined by the workflow owner by assigning an integer value to the counter *Number of Automatic Restarts*.
+The user also defines *Where he wants that the scheduler restarts In-Error tasks*, i.e., on the same proactive node or on another proactive node.
+
+If automatic restarts does not fix errors, the scheduler applies one of the following task error management policies, i.e., the one defined by the user during workflow creation:
```

Comment:
Based on README file available at the root of the project:

> Use capitals for ProActive’s terms, as defined in the glossary
> 
> Workflow, Job, Scheduler
> 

scheduler -> Scheduler 

same apply for Task, etc.